### PR TITLE
fixes ranking and adds valleB

### PR DESCRIPTION
**Pull request by Kaffe-work**
fixes the last bug in ranking(probably) adds ValleB css, removes styrit css (for now), adds kandidatmiddagsgrupp.

### CODE REVIEW

File: app/controllers/stats_controller.rb [hunk 1]
```diff
@@ -53,7 +53,7 @@ def hours
     if params[:user_id]
       user_id = params[:user_id]
       cache_key = "hours/#{user_id}"
-      query = query.with_user(user_id)
+      query = query.with_user(user_id)
```
Comment (malmers) [inline app/controllers/stats_controller.rb:56]: Trailing space

File: app/assets/stylesheets/sessions.css.scss [hunk 2]
```diff
@@ -108,17 +108,19 @@
...
+  tr#valberedningen {
+    background: image-url("valleb.png");
     th {
+      //padding: 1px;
+      text-align: end;
       color: white;
     }
   }
+
 }
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss:126]: No new-line at end of file

File: app/assets/stylesheets/sessions.css.scss [hunk 3]
```diff
@@ -108,17 +108,19 @@
...
+  tr#drawit {
+    background: image-url("drawit.png");
     th {
       color: white;
-      text-shadow: -1px 0 black, 0 1px black, 1px 0 black, 0 -1px black;
     }
-  }
-  tr#drawit {
-    background: image-url("drawit.png");
+  }
```
Comment (malmers) [inline app/assets/stylesheets/sessions.css.scss:116]: Random space

File: app/models/user.rb [hunk 4]
```diff
@@ -1,3 +1,111 @@
...
+			        'Revisorerna'
+			when :dpo
+							'DPO'
+			when :kandidatmiddagen
+							'Kandidatmiddag'
+			else
+				self.to_s
+		end
+	end
+=======
```
Comment (malmers) [inline app/models/user.rb:108]: Wait what

File: app/views/stats/show.html.erb [hunk 5]
```diff
@@ -1,6 +1,7 @@
-<h1><%= @user.nickname %></h1>
-<%= link_to "Position: #{@ranking}", all_stats_path(anchor: @user.id) %><br/>
-<%= link_to "All time position: #{@year_ranking}", all_stats_path(anchor: @user.id, timeframe: "all_time") %><br/>    
+
+<h1><%= @user.nick %></h1>
```
Comment (malmers) [inline app/views/stats/show.html.erb:2]: @user.nickname is the correct method post OAuth2

**Review by malmers — COMMENTED** [review]
This PR sure does a lot of different things

File: app/views/stats/show.html.erb (hunk 5 above)
Comment (lindskogen) [inline app/views/stats/show.html.erb:2]:
```suggestion
<h1><%= @user.nickname %></h1>
```

**Pull request closed without merging**